docs/PRIVATE.md

```diff
@@ -0,0 +1,225 @@
+## Modules
+
+<dl>
+<dt><a href="#module_deriver-DeriverUtils">deriver-DeriverUtils</a> ℗</dt>
+<dd><p>Scoped key deriver utilities</p>
+</dd>
+<dt><a href="#module_deriver-ScopedKeys">deriver-ScopedKeys</a></dt>
+<dd><p>Used by the Firefox Accounts content server</p>
+</dd>
+<dt><a href="#module_relier-KeyUtils">relier-KeyUtils</a> ℗</dt>
+<dd><p>Scoped key utilities</p>
+</dd>
+<dt><a href="#module_relier-OAuthUtils">relier-OAuthUtils</a></dt>
+<dd><p>OAuth methods to authenticate with Firefox Accounts and get scoped keys</p>
+</dd>
+<dt><a href="#module_relier-util">relier-util</a> ℗</dt>
+<dd><p>Relier utility functions</p>
+</dd>
+</dl>
+
+<a name="module_deriver-DeriverUtils"></a>
+
+## deriver-DeriverUtils ℗
+Scoped key deriver utilities
+
+**Access**: private  
+**Example**  
+```js
+
+```
+<a name="module_deriver-DeriverUtils..encryptBundle"></a>
+
+### deriver-DeriverUtils~encryptBundle(appPublicKeyJwk, bundle) ⇒ <code>Promise</code>
+**Kind**: inner method of [<code>deriver-DeriverUtils</code>](#module_deriver-DeriverUtils)  
+
+| Param | Type |
+| --- | --- |
+| appPublicKeyJwk | <code>string</code> | 
+| bundle | <code>object</code> | 
+
+<a name="module_deriver-ScopedKeys"></a>
+
+## deriver-ScopedKeys
+Used by the Firefox Accounts content server
+
+**Example**  
+```js
+const scopedKeys = new fxaCryptoDeriver.ScopedKeys();
```

Comment:
Out of curiosity, why do we need to use `new` to create an instance of this?  It doesn't look like the API is stateful at all.

Reply:
In the relier part there is a `jose.JWK.createKeyStore()` that has state. This is for consistency, but we can change that